package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "5.30.0",
+  "version": "5.31.0",
```

Comment:
Wondering if this could be classed as a patch i.e. 5.30.1?
Looking at https://semver.org/ they say 'Bug fixes not affecting the API increment the patch version, backwards compatible API additions/changes increment the minor version, and backwards incompatible API changes increment the major version.'. I think this could be classed as an accessiblity bug fix as it doesn't affect the API in any way?